squad-server/plugins/basic-stats.js

```diff
@@ -0,0 +1,736 @@
+import BasePlugin from './base-plugin.js';
```

Comment:
Run a yarn lint and fix the errors present please. 

```
yarn lint
yarn run v1.22.22
$ eslint --fix . && prettier --write "./**/*.js"

/workspaces/SquadJS/squad-server/plugins/basic-stats.js
  365:32  error  Expected '!==' and instead saw '!='       eqeqeq
  380:67  error  Expected '===' and instead saw '=='       eqeqeq
  380:99  error  Expected '===' and instead saw '=='       eqeqeq
  651:15  error  'tks' is assigned a value but never used  no-unused-vars
  707:15  error  'rpd' is assigned a value but never used  no-unused-vars
  708:15  error  'hpr' is assigned a value but never used  no-unused-vars

✖ 6 problems (6 errors, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Fixed